internal/services/compute/disk_encryption_set_resource_test.go

```diff
@@ -101,16 +94,6 @@ func TestAccDiskEncryptionSet_keyRotate(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.ImportStep(),
-		// we have to first grant the permission for DiskEncryptionSet to access the KeyVault
-		{
-			Config: r.grantAccessToKeyVault(data),
-			Check: acceptance.ComposeTestCheckFunc(
-				check.That(data.ResourceName).ExistsInAzure(r),
-			),
-		},
-		data.ImportStep(),
```

Comment:
This part is used to add the below resource, and it is moved to the `dependencies` function, and will be present when `basic` step is run
```
resource "azurerm_key_vault_access_policy" "disk-encryption" {
  key_vault_id = azurerm_key_vault.test.id
  key_permissions = [
    "Get",
    "WrapKey",
    "UnwrapKey",
  ]
  tenant_id = azurerm_disk_encryption_set.test.identity.0.tenant_id
  object_id = azurerm_disk_encryption_set.test.identity.0.principal_id
}
```